cfpb/cfgov-refresh
# Add categories for Reporting Companies List

This adds a "Reporting Companies List" `page_type` to `refs.py`, allowing a new filterable list page type with a new set of categories.

## Additions
- Consumer Reporting Companies and related categories added to `ref.py`
- New migration created

## Testing
1. Pull down the branch
2. Run the migration
3. Go into Wagtail (make sure you have recent data) and check out the Consumer Reporting Companies page. Be sure to find some of the subpages, add categories to them, and publish them. Then publish and check out the Consumer Reporting Companies page and play with the filterable list!

## Checklist

## Testing checklist

### Browsers


### Accessibility


### Other
